src/pages/index.js

```diff
@@ -174,15 +170,18 @@ function handleAddCardFormSubmit(cardData) {
     });
```

Comment:
`cardDeleteModal.setLoading(true);`


You need to start loading **before** sending a request to see the button text chnaging